src/components/navbar/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { forwardRef } from 'react'
+import PropTypes from 'prop-types'
+
+import NavbarHeader from './navbar-header'
+import NavbarItem from './navbar-item'
+
+
+const Navbar = forwardRef(({ children, classes, ...rest }, ref) => {
+  const navbarClasses = Object.freeze({
+    root: `${classes.root} navbar__root w-screen h-16 px-10 flex items-center justify-between`,
+  })
+  return (
+    <div
+      ref={ref}
+      className={navbarClasses.root} 
+      {...rest}
+    >
+      {React.Children.map(children, (child) => {
+        if (React.isValidElement(child) && child.type.__docgenInfo.displayName === 'NavbarHeader') {
+          return React.cloneElement(child)
+        }
+      })}
+
+      <div className="navbar__items flex items-center justify-center">
```

Comment:
I would add here a `navbarClasses.items` class so we can customize this as we wish.